redisless/src/server.rs

```diff
@@ -174,6 +174,14 @@ fn run_command_and_get_response<T: Storage>(
                 unlock(storage).set(k.as_slice(), v.as_slice());
                 protocol::OK.to_vec()
             }
+            Command::Setex(k, v, duration) => {
+                unlock(storage).setex(k.as_slice(), v.as_slice(), *duration);
+                protocol::OK.to_vec()
+            }
+            Command::Expire(k, duration) => {
+                unlock(storage).expire(k.as_slice(), *duration);
```

Comment:
Hey :) We should follow Redis API (for the return type for EXPIRE command here) - https://redis.io/commands/expire#return-value